app/services/slack_notifier_service.rb

```diff
@@ -0,0 +1,34 @@
+class SlackNotifierService
```

Comment:
This is very much personal opinion but I'd suggest rather then having this as a static class, ie

```
SlackNotifierService.call
```

instead differentiate between 'action' data and maintained state data, eg

```
SlackNotifierService.new(msg_notifier).notify('my message to slack')
```

Reply:
it sounds good to me, my choice was influenced by the way we do services. I'll give it a try.

Reply:
I made some changes, now the notifier is stored in the Rails.application.config.slack_notifier.